### ISSUE TITLE
5% of BA.1.1 carrying the R346K mutations are classified as BA.1

### ISSUE THREAD

**Issue by wodanaz** [cov-lineages/pangoLEARN#80]
Hi folks,

I posted this issue in pangolin and realized that it was the wrong place. Sorry about that.

In our genomic surveillance data, we have been consistently finding that about 5% of samples with high quality sequencing seem to be misclassified as BA.1 even when using the most recent version of pangolin and pangoLEARN. In addition, almost all the misclassified samples BA.1s have a phylogenetic backbone that corresponds to BA.1.1 and they have great calling quality for the R346K mutation. Given the medical importance of that mutation (R346K) I hope this phenomenon can be reviewed .

Example of read depth and quality in one sample

![image](https://user-images.githubusercontent.com/5439367/161263241-a54e16e6-7ab6-4d1e-b0d0-e3a2603e495c.png)

and Phylogeny:

and an example of the tree placing of the BA.1 (These samples are confirmed to have the R346K mutation)

![image](https://user-images.githubusercontent.com/5439367/161263296-535a7a3f-17e4-44f3-8277-85b61819c54d.png)

Thank you so much an keep up with the great work

Warmly,

Alejandro Berrio
Duke University

**Comment by wodanaz** [comment]
Fantastic, thank you!

**Comment by aineniamh** [comment]
I believe adding more representatives into the designations will resolve the pangoLEARN model error, that's how the pangoLEARN issues are usually resolved (the decision tree is definitely over-fit, and the more informative training data it gets the better it does), this is why I suggested porting the issue over to designation. pangolin 4.0 has a new model which is a random forest, which should be less overfit and give more interpretable confidence scores too. Hopefully the usher mode will resolve this all anyway as pangolin 4.0 has just been released. 

FYI @corneliusroemer it's not an issue for the pangolin repo, as it's not pangolin software related just for future ref. I appreciate your great explanation above though! 

@wodanaz if you still see issues with the assignments with the latest pangolin version, modes and models please let me know and I'll get to the bottom of it! If there's something particularly tricksy going on we might need to add a constellation definition in.

**Comment by corneliusroemer** [comment]
@aineniamh you're right, I reworded slightly. It was originally in pango-designation but it belongs only here in pangoLEARN, not pangolin.
